### ISSUE TITLE
missing sdl headers

### ISSUE THREAD

**Issue by malytomas** [janhsimon/Makma#3]
sdl2.dll is part of the repository while sdl.h is not.
(building with visual studio 2017)

**Comment by janhsimon** [comment]
Oops, good catch. SDL2.dll (and assimp.dll) shouldn't be part of the repository. The project needs to be built against SDL, assimp and the Vulkan SDK.

I *could* copy SDL and assimp headers into the project but that seems bad form, and doing the same for the Vulkan SDK is not exactly an option... The other dependencies were on GitHub already so I could add them as external submodules, which seems to be the most elegant option.

So, either I add the SDL and assimp headers to the project so it's set up to work out of the box BUT then you still need to add the Vulkan SDK. Or I remove the SDL2.dll and assimp.dll files.

Any preferences?

**Comment by malytomas** [comment]
If you were using cmake, it would be easy to integrate them as submodules.
SDL is available at https://github.com/spurious/SDL-mirror (that is what I use).
Assimp: https://github.com/assimp/assimp.git
Is it difficult to _inject_ cmake dependencies into meson?

Either way, the dlls should probably be removed. They may be incompatible based on build settings.

**Comment by janhsimon** [comment]
I don't use meson, that was added by a Linux user (and the script is now outdated), I'm just on Visual Studio. For now I've added the DLLs and added a big red note to the readme that mentions the necessity to build the project against SDL, assimp and the LunarG Vulkan SDK (to make the necessary local changes to the Visual Studio project and copy the two DLLs).

This is the best I can do for now, so I'm going to close the issue. But further input/discussion is always welcome!
